Source/Atomic/IO/PackageFile.cpp

```diff
@@ -166,12 +166,18 @@ void PackageFile::Scan(Vector<String>& result, const String& pathName, const Str
     if (filterExtension.Contains('*'))
         filterExtension.Clear();
 
+    bool caseSensitive = true;
+#ifdef _WIN32
+    // On Windows ignore case in string comparisons

```

Comment:
In general like to stick with ATOMIC_PLATFORM_WINDOWS for windows specific code, just noting, don't need adjustment for this PR